python_modules/dagster/dagster/_core/storage/tags.py

```diff
@@ -57,7 +57,9 @@
 RETRY_NUMBER_TAG = f"{SYSTEM_TAG_PREFIX}retry_number"
 RETRY_STRATEGY_TAG = f"{SYSTEM_TAG_PREFIX}retry_strategy"
 
-USER_EDITABLE_SYSTEM_TAGS = [PRIORITY_TAG, MAX_RETRIES_TAG, RETRY_STRATEGY_TAG]
+MAX_RUNTIME_TAG = f"{SYSTEM_TAG_PREFIX}max_runtime"
```

Comment:
this is seconds right? Will people assume that? max_runtime_seconds is kind of a mouthful but is an option

Reply:
yea I kind of like the explicit-ness of it (and yes it is in seconds)